### PR TITLE
New Blog Post: Tickets are on sale post

### PR DESCRIPTION
**Pull request by gnunicorn**

### CODE REVIEW

File: _posts/2017-02-25-tickets-on-sale-and-timeline.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+We hadn't fully announced the new website yet, and people already asked about when they can buy the tickets. Well, you are in luck, because as off today you can 
+
+[![Get your Ticket now (block of tickets with prices from website)](/assets/posts/2017-tickets.png)](http://2017.rustfest.eu/)
+
+You can find these on the main home page.
+
+### Timeline and prices
+
+The same as last year, we are starting with offering our [✨ Supporter ✨ Ticket](https://ti.to/asquera-event-ug/rustfest-kyiv/with/nhv7rcouqe0) first (from 199€+ _- available now_) and will sell Regular Tickets in batches again. In order to reward who is willing to commit early, the prices will increase with every batch. They will go on sale as follows:
```
Comment (flaki) [inline _posts/2017-02-25-tickets-on-sale-and-timeline.md:18]: One nit, why does this URL include a coupon? (or, at least, I think it's a coupon)

The ticket seems to show up without it, anyhow: https://ti.to/asquera-event-ug/rustfest-kyiv/

**Comment by flaki** [comment]
Besides the one nit about the Supporter ticket URL, this LGTM!

File: _posts/2017-02-25-tickets-on-sale-and-timeline.md (hunk 1 above)
Comment (gnunicorn) [reply _posts/2017-02-25-tickets-on-sale-and-timeline.md:18]: not a coupon, just the "official ticket url" (given from Ti.to). I think those are also used to link to specific hidden tickets but secondly, it highlights the Ticket in question and puts the focus in the right field, once there are more than just one ticket. I'd leave it in, doesn't hurt.

**Comment by gnunicorn** [comment]
as @flaki reviewed, you'll become official second author :D

**Pull request merged**